staging/src/k8s.io/client-go/transport/token_source.go

```diff
@@ -182,6 +182,8 @@ func (ts *cachingTokenSource) Token() (*oauth2.Token, error) {
 		if ts.tok == nil {
 			return nil, err
 		}
+		// TODO: avoid logging here - but not sure how.
```

Comment:
connect TODOs to an issue we can go to for context / assignment / tracking

Reply:
Or remove this TODO and decide now that we just leave this call as it is?

If I were to create an issue, it would probably never get addressed and just cause constant work for re-triaging until eventually it gets auto-closed.

Reply:
fine with me

Reply:
After thinking about this again, I used `klog.TODO().Error(err, "Unable to rotate token")` with a long comment explaining why there is no tracking issue.

I could have used `klog.Background()`, but that isn't quite right: if someone finds a way to pass in a logger, that line should be changed. I've encountered quite a few places where developers used `context.Background()` in places where passing in a context would have been the right thing to do; this here would be similar.